desktop/wrapper/src/handle_desktop_wrapper_message.rs

```diff
@@ -118,5 +118,22 @@ pub(super) fn handle_desktop_wrapper_message(dispatcher: &mut DesktopWrapperMess
 			let message = AppWindowMessage::AppWindowUpdatePlatform { platform };
 			dispatcher.queue_editor_message(message.into());
 		}
+		DesktopWrapperMessage::LoadDocument { id, document, to_front } => {
+			let message = PortfolioMessage::OpenDocumentFileWithId {
+				document_id: id,
+				document_name: Some(document.name),
+				document_path: document.path,
+				document_serialized_content: document.content,
+				document_is_auto_saved: true,
+				document_is_saved: document.is_saved,
+				to_front,
+				select_after_open: false,
```

Comment:
This is the rebase conflict I fixed, since I introduced this `select_after_open` field in a PR I added yesterday. You may want to check if `false` is appropriate here, or if you want to feed this in as a variable (like `to_front` on the line above gets fed in as a variable) so callers of this message can decide on a case-by-case basis.